pkg/cstor/cstor.go

```diff
@@ -268,6 +274,11 @@ func (p *Plugin) Init(config map[string]string) error {
 
 	if restoreAllSnapshots, ok := config[RestoreAllIncrementalSnapshots]; ok && restoreAllSnapshots == "true" {
 		p.restoreAllSnapshots = true
+		p.autoSetTargetIP = true
+	}
+
+	if autoSetTargetIP, ok := config[AutoSetTargetIP]; ok {
```

Comment:
Here we are checking if value is set or not. We also need to check if `autoSetTargetIP` is set to "true".

Reply:
the value is set like this:
```golang
p.autoSetTargetIP = autoSetTargetIP == "true"
```

so if the value is "true" it will be true, false otherwise.
I did it in this way, because restoreAllIncrementalSnapshots="true" implies autoSetTargetIP="true" and wanted to provide a way to turn it off.

so now if restoreAllIncrementalSnapshots="true" and autoSetTargetIP="false"  the targetip won't be set